### PR TITLE
Refactor node dumping to avoid extra code in Infer method

### PR DESCRIPTION
**Pull request by EgorDuplensky**
### Details:
- Clean up code after recent Infer() updates

### Tickets:
 -

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp [hunk 1]
```diff
@@ -422,18 +423,20 @@ void MKLDNNGraph::ExecuteConstantNodesOnly() {
...
-                graphNode->execute(stream);
+                DUMP(node);
+                node->execute(stream);
 
                 for (auto & output : std::get<2>(sharedOutputs))
                     output->valid(true);
             }
         } else {
-            graphNode->execute(stream);
+            DUMP(node);
```
Comment (EgorDuplensky) [inline inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp:438]: I propose to create "executeNode" method for graph:
```c++
inline void executeNode(node) {
        DUMP(node, infer_count);
        OV_ITT_SCOPED_TASK(itt::domains::MKLDNNPlugin, node->profiling.execute);
        node->execute(stream);
}
```
Not sure if we need ITT counters for constant node though

Comment (dmitry-gorokhov) [reply inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp:438]: Agree. Lets create. We need ITT here as well for FIL analysis

**Review by dmitry-gorokhov — APPROVED** [review]
LGTM in general

**Comment by EgorDuplensky** [comment]
@dmitry-gorokhov ExecuteNode method has been added

**Comment by dmitry-gorokhov** [comment]
@EgorDuplensky Please rebase on latest master and I will merge it.

**Pull request merged**